data/ecosystems/z/zksync.toml

```diff
@@ -4811,6 +4816,3 @@ url = "https://github.com/zy19900820/ethers"
 
 [[repo]]
 url = "https://github.com/zycta/zycta-dapp"
-
-[[repo]]
-url = "https://github.com/dappsheriff/zksync-contracts"
```

Comment:
we prefer to keep archived / missing repos because we have a persisted history that we can attribute to the parent ecosystem, please revert this deletion